### PR TITLE
Add usage with URL

### PR DESCRIPTION
**Pull request by sundowndev**
### References

- https://github.com/cloudskiff/driftctl/pull/335

### CODE REVIEW

File: docs/usage/cmd/scan/iac-source.mdx [hunk 1]
```diff
@@ -29,13 +29,17 @@ $ driftctl scan --from tfstate+s3://statebucketdriftctl/states
...
+# We also support fetching tfstate manifests from URLs
+$ driftctl scan --from tfsate+https://example.com/terraform.tfsate
 ```
 
 ## Supported IaC sources
 
 - Terraform state
   - Local: `--from tfstate://terraform.tfstate`
   - S3: `--from tfstate+s3://my-bucket/path/to/state.tfstate`
+  - HTTP `--from tfsate+http://my-url/state.tfsate`
```
Comment (wbeuil) [inline docs/usage/cmd/scan/iac-source.mdx:42]: `HTTP:` and `tfstate`

File: docs/usage/cmd/scan/iac-source.mdx [hunk 2]
```diff
@@ -29,13 +29,17 @@ $ driftctl scan --from tfstate+s3://statebucketdriftctl/states
 # N.B. Symlinks under the root folder will be ignored.
 #      If the folder itself is a symlink it will be followed.
 $ driftctl scan --from tfstate://my-states/directory
+
+# We also support fetching tfstate manifests from URLs
+$ driftctl scan --from tfsate+https://example.com/terraform.tfsate
```
Comment (wbeuil) [inline docs/usage/cmd/scan/iac-source.mdx:34]: `tfstate`

**Review by wbeuil — CHANGES_REQUESTED** [review]

File: docs/usage/cmd/scan/iac-source.mdx (hunk 2 above)
Comment (sundowndev) [reply docs/usage/cmd/scan/iac-source.mdx:34]: Not sure what you're referring to, do you mean `driftctl scan --from tfsate://example.com/terraform.tfsate` ?

Comment (wbeuil) [reply docs/usage/cmd/scan/iac-source.mdx:34]: You wrote `tfsate` instead of `tfstate` everywhere :)

Comment (sundowndev) [reply docs/usage/cmd/scan/iac-source.mdx:34]: Oh right, thanks :smile:

**Review by wbeuil — COMMENTED** [review]
I'm good with that for the moment, we should wait for your second part on header flags to go further with this PR.

**Review by wbeuil — APPROVED** [review]
LGTM

**Pull request merged**